### PR TITLE
Add command option to only show selected squad player's replay elims

### PR DESCRIPTION
**Pull request by chao813**

### CODE REVIEW

File: fortnite_bot.py [hunk 1]
```diff
@@ -257,40 +257,68 @@ async def replays_operations(ctx, *params):
...
         return
 
-    command = params.pop(0) if params else None
+    command = None
+    username = None
+    if len(params) == 2:
+        username = params.pop(1)
+        command = params.pop(0) 
+        if username not in SQUAD_PLAYERS_LIST:
+            await ctx.send(f"{username} provided is not a valid squad player")
```
Comment (kwkevinlin) [inline fortnite_bot.py:266]: Do you want to return after this?

Comment (chao813) [reply fortnite_bot.py:266]: oh yeah ur right, should I just `return`

File: fortnite_bot.py [hunk 2]
```diff
@@ -257,40 +257,68 @@ async def replays_operations(ctx, *params):
...
 
-async def output_replay_eliminated_by_me_stats_message(ctx, eliminated_by_me_dict, silent):
+async def output_replay_eliminated_by_me_stats_message(ctx, eliminated_by_me_dict, username, silent):
     """ Create Discord Message for the stats of the opponents that got eliminated by us"""
-    for squad_player in eliminated_by_me_dict:
+    if username:
         if not silent:
-            await ctx.send(f"{squad_player} eliminated")
-        for player_guid in eliminated_by_me_dict[squad_player]:
+            await ctx.send(f"{username} eliminated")
```
Comment (kwkevinlin) [inline fortnite_bot.py:313]: not: there's duplication here with the below which can be combined

File: fortnite_bot.py [hunk 3]
```diff
@@ -257,40 +257,68 @@ async def replays_operations(ctx, *params):
...
-        squad_players_eliminated_by_player = ""
-        for squad_player in eliminated_me_dict[player_guid]:
-            squad_players_eliminated_by_player += squad_player + ", "
-        if not silent:
-            await ctx.send(f"Eliminated {squad_players_eliminated_by_player[:-2]}")
-        await player_search(ctx, player_guid, guid=True, silent=silent)
+    if username:
+        for player_guid in eliminated_me_dict:
+            if username in eliminated_me_dict[player_guid]:
+                if not silent:
```
Comment (kwkevinlin) [inline fortnite_bot.py:296]: nit: The other duplication, only diff is ctx.send message

Usually I follow a "if 3 times, then merge" idea, but this is so close together I think it's worth it to combine

File: clients/fortnite_api.py [hunk 4]
```diff
@@ -35,7 +35,8 @@ async def get_player_stats(ctx, player_name, guid):
             result = await _get_player_account_id(session, player_name, "psn")
             stats = await _get_player_stats(session, account_id)
             if stats["global_stats"] is None:
-                await ctx.send("{player_name}'s statistics are hidden".format(player_name=stats["name"]))
+                stats_hidden_embed=discord.Embed(title="{player_name}'s statistics are hidden".format(player_name=stats["name"]))
```
Comment (kwkevinlin) [inline clients/fortnite_api.py:38]: nit: f strings preferred for consistency

**Pull request merged**